src/components/step-nav.scss

```diff
@@ -10,46 +10,48 @@
 // * 5. Titled Modifier
 
 // TODO : document global dependencies
-// TODO : Add Titled Modifier
+// TODO : further responsive cleanup, especially on titled setp nav
 
 //------------------------------------------------------------//
 
 // * 1. Settings
 
 $StepNav-size: 60px;
 $StepNav-border: 4px;
+$StepNav-paddedWidth: $StepNav-size + ($StepNav-border * 2);
```

Comment:
Nit: This threw me off a little... since `$StepNav-border` is used to define both the border and the padding. In this case, it's referring to the border value as _padding_, correct? I wonder if adding a variable like `$StepNav-padding: $StepNav-border` might make this clearer.